pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -61,7 +63,9 @@ const (
 
 	// reconcilerSyncDuration is the amount of time the reconciler sync states loop
 	// wait between successive executions
-	reconcilerSyncDuration time.Duration = 5 * time.Second
+	// This has been increased from every 5 seconds to every 5 minutes since
+	// the timing has created an enormous amount of API traffic on such clouds as AWS.
+	reconcilerSyncDuration time.Duration = 5 * time.Minute
```

Comment:
Let's make this 1 minute instead of 5 (which will still reduce the rate 12x times). There is a tradeoff--without this check we could potentially reintroduce the AWS bug of wrong volume being attached.

Also maybe make this value configurable/enabled/disable in the same PR?